pytorch_lightning/core/lightning.py

```diff
@@ -1386,16 +1241,10 @@ def optimizer_step(self, epoch, batch_idx, optimizer, optimizer_idx,
         """
         if on_tpu:
```

Comment:
In a follow on PR, can we use self.trainer.accelerator_backend.backward(...)?

Reply:
I am not sure why you want to use `.backward` on `.step`. manual_backward is the one calling `.backward`.